ray-operator/controllers/ray/common/constant.go

```diff
@@ -72,7 +72,7 @@ const (
 
 	// Use as container env variable
 	NAMESPACE                               = "NAMESPACE"
-	CLUSTER_NAME                            = "CLUSTER_NAME"
+	RAY_CLUSTER_NAME                        = "RAY_CLUSTER_NAME"
```

Comment:
`KUBERAY_RAYCLUSTER_NAME` or `KUBERAY_CLUSTER_NAME`? 

It is very easy for Ray community to use the environment variable with the name `RAY_CLUSTER_NAME`.